Tests/SystemTests/PublisherSubscriber.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+
+class PublisherSubscriber: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testExample() throws {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    }
+
+    func testPerformanceExample() throws {
+        // This is an example of a performance test case.
+        self.measure {
+            // Put the code you want to measure the time of here.
+        }
+    }
+
+}
```

Comment:
empty test case - remove it

Reply:
Different to the other one you saw (connectionConfigurationTests). this is one is there because I needed a test file in `SystemTests` module/ test target to actually run this new test target I created. Its empty currently, but I am using it in the Token Authentication feature I am adding. It will only be temporary for this one PR.

Otherwise I got `xcodebuild: error: The workspace named "AblyAssetTracking" does not contain a scheme named "SystemTests"`, because Xcode doesn't generate a Scheme for a test target without any source files.

However, I have **simplified** it in ce627dcea504709b2a45355510e1cfa2c2992eac